src/main/haskell/language-kore/src/Data/Kore/ASTVerifier/ModuleVerifier.hs

```diff
@@ -1,62 +1,39 @@
-module Data.Kore.ASTVerifier.ModuleVerifier (verifyModule,
-                                             verifyUniqueNames) where
+module Data.Kore.ASTVerifier.ModuleVerifier
+    (verifyModule, verifyUniqueNames) where
```

Comment:
any reason to not follow existing conventions for indenting this?

Reply:
None, sorry.